doc/manual/manual.tex

```diff
@@ -2286,6 +2286,58 @@ \subsection{Advection Stabilization}
 \end{itemize}
 
 
+Both add additional terms to the temperature (or compositional field equation). We will discuss
+the case for the temperature equation here. The compositional fields only differ in having
+a zero conductivity, fewer right-hand side terms, and $\rho C_p=1$. The strong form of the temperature equation reads
+\[
+ \rho C_p \frac{\partial T}{\partial t} + \rho C_p \mathbf{u} \cdot \nabla T - \nabla \cdot k\nabla T = F,
+\]
+where $F$ is the combination of source and reaction terms,
+while the weak form (with test function $\varphi$) becomes:
+\[
+a(T,\varphi) =
+ \left(\rho C_p \frac{\partial T}{\partial t}, \varphi \right)
+ + \left(\rho C_p \mathbf{u} \cdot \nabla T, \varphi \right)
+ + \left( k \nabla T, \nabla \varphi \right) = (F,\varphi) = f(\varphi).
+\]
```

Comment:
There is only one other place in the manual where we write about the weak form (in the inner core cookbook), and there we write out the integrals. So if you want to use this shorter form here, it would make sense to define what you mean by that.